### DISCUSSION TITLE
[Question][lecture0]

### DISCUSSION THREAD

**Discussion by x14ngch3n** [prosyslab-classroom/cs524-program-analysis#340] [Q&A] [answered]
Name: cascades-sjtu

Hi, there's a mistake in the snippet code in [lecture0.pdf](https://github.com/prosyslab-classroom/cs524-program-analysis/blob/main/slides/lecture0.pdf) page18. The result should be 18

```ocaml
# add_high (fun x -> x + 5) 10;;
- : int = 17
```

**Comment by KihongHeo** [comment, marked as answer]
Hi. Thanks. Fixed.
